docs/Legacy-Configuration.md

```diff
@@ -0,0 +1,420 @@
+## Mapping with JSON Path
+
+The two device content template types supported today rely on JSON Path to both match the desired template and extract values.  
+More information on JSON Path can be found [here](https://goessner.net/articles/JsonPath/). Both template types use the [JSON .NET implementation](https://www.newtonsoft.com/json/help/html/QueryJsonSelectTokenJsonPath.htm) for resolving JSON Path expressions. Additional examples can be found in the [unit tests](../test/Microsoft.Health.Fhir.Ingest.UnitTests/Template/JsonPathContentTemplateTests.cs).
```

Comment:
Now that we have more than 2 device content templates, we probably need to update the first sentence - It looks like we have 3 defined in this file and 1 new type that does not 'rely' on JSON Path, but can use it.

Reply:
Thinking about it more, I think that that I'll create separate pages for each Template Type. I can then link to them from the main Configuration page with a statement of what each one is and why'd you'd want to use it.